FALSEHOODS.md

```diff
@@ -235,3 +235,10 @@ surprising how many false assumptions programmers continue to make about them.
     during, or after the actual implementation of numbering plan changes in the
     real world. The actual date on which a phone number range becomes active may
     not always adhere to official announcements.
+
+1.  **Phone numbers are numbers**
+
+    Never try to store phone numbers in an int or any other kind of numeric
+    data type. You can't do arithmetic on them, and while 007, 07 and 7 are the
+    same number they are not necessary the same phone number. In some countries
+    a leading 0 is significant and forms part of the number itself.
```

Comment:
The last sentence should link to the falsehood we already have - A leading zero in numbers formatted for domestic usage can always be discarded when dialing from abroad

Also you should say "Moreover, a phone number may contain other diallable characters (see Phone numbers contain only digits) or an extension portion, dialled after waiting for a tone.

Without these points, I think people could think that storing the number in E164 format would work, since no phone number country calling code starts with a 0, and the + can be assumed.